nilearn/input_data/base_masker.py

```diff
@@ -145,6 +144,13 @@ def transform_single_imgs(self, imgs, confounds=None, copy=True):
             documentation for details.
             shape: (number of scans, number of confounds)
 
+        sample_mask : Any type compatible with numpy-array indexing, optional
+            shape: (number of scans - number of volumes removed, )
+            Masks the niimgs along time/fourth dimension to perform scrubbing
+            (remove volumes with high motion) and/or non-steady-state volumes.
+            This parameter is passed to signal.clean.
+                .. versionadded:: 0.7.2
```

Comment:
I think you need a blank line before this line, otherwise Sphinx will give a warning which will be treated as an error when building the doc in strict mode. 

```
...
This parameter is passed to signal.clean.

    .. versionadded:: 0.7.2
```